addons/mail/static/src/messaging/entity/activity/activity.js

```diff
@@ -41,6 +41,14 @@ function ActivityFactory({ Entity }) {
             }
         }
 
+        /**
+         * @returns {boolean}
+         */
+        get isCurrentPartnerAssignee() {
+            return this.assignee &&
+                this.assignee.partner  === this.env.messaging.currentPartner;
```

Comment:
strange indent and double space before ===
maybe wrap around ( ) and make one condition per line

or I think often we just do a preemptive return if `!this.assignee` and then just return the simple condition if ok